### ISSUE TITLE
Support for some older modules like VMB4RF, VMB1BLS and VMBPIRO

### ISSUE THREAD

**Issue by d3claes** [thomasdelaet/python-velbus#66]
Hi, this weekend I've explored a setup for Home Assistant (v. 0.116.2) and Velbus. 
I have a working openHAB + velbus setup, but due to high memory usage problems every few weeks, I wanted to try HA instead.
All my Velbus modules are automatically recognized (thank you community!).
Except for:

* [VMB4RF](https://www.velbus.eu/downloads/0/velbus/manuals/protocol/protocol_vmb4rf.pdf) (used for opening/closing of a gate)
* [VMB1BLS](https://www.velbus.eu/downloads/0/velbus/manuals/protocol/protocol_vmb1bls.pdf) (used for two blinds next to each other)

Although not in the list of supported modules, the module [VMBPIRO](https://www.velbus.eu/downloads/0/velbus/manuals/protocol/protocol_vmbpiro.pdf) is recognized with one channel 9 (outside temperature).

I've no coding experience for these kind of py-scripts. Maybe someone can help with one of these modules? Maybe this is possible for VMB1BLS, because there are already some blind-modules supported.

Thank you for more information.
Kinds regards

**Comment by Cereal2nd** [comment]
as i don't own the modules its hard to implement/test these ...

for the vb4rf:
compare this command reference to the one for a vmb2pbn and see whtas different in the following messages
1. PushButtonStatusMessage
2. ModuleStatusMessage
if the differences are not to big it could be an easy implemntation

for the vmb1bls
same as above but compare to the VMB1BL
1. BlindStatusMessage

This is the first step, depending of the outcome it can be a really simple implementation

**Comment by d3claes** [comment]
Hi,

Sure, I want to compare these command references, but what do I need to do? I'm fumbling a bit in the dark.

Thank you for more information.
Kind regards.

**Comment by Cereal2nd** [comment]
the packet format,if its different it will be more work, if its the same it can go fast and is easy

**Comment by d3claes** [comment]
Hi,

I'm afraid that I don't have the technical knowledge to master the velbus system. I'm helping out my father who wants to have a display in Home Assistant, because the buttons on his VMBLCDWB are confusing him from time to time.

I've made some screenshots of the commando's up en down for the VMB1BLS and VMB1BL.

**VMB1BLS-UP**
![VMB1BLS_up](https://user-images.githubusercontent.com/2638856/96122408-bc0fee80-0ef1-11eb-9930-c1ba0a782f9f.png)

**VMB1BLS-DOWN**
![VMB1BLS_down](https://user-images.githubusercontent.com/2638856/96122429-c16d3900-0ef1-11eb-9167-ef8dcff0f5be.png)

**VMB1BL-UP**
![VMB1BL_up](https://user-images.githubusercontent.com/2638856/96122444-c7fbb080-0ef1-11eb-8903-b13bc0338f3d.png)

**VMB1BL-DOWN**
![VMB1BL_down](https://user-images.githubusercontent.com/2638856/96122459-cd58fb00-0ef1-11eb-88a6-13abbf04b562.png)

If I compare the protocol of the two modules in their pdf, they have one difference:
**VMB1BLS**
<SOF-SID10...SID0-RTR-IDE-r0-DLC3...0-DATABYTE1...DATABYTEn-**CRC14**...CRC1-CRCDEL-ACK-ACKDELEOF7...EOF1-IFS3...IFS1>
**VMB1BL**
<SOF-SID10...SID0-RTR-IDE-r0-DLC3...0-DATABYTE1...DATABYTEn-**CRC15**...CRC1-CRCDEL-ACK-ACKDELEOF7...EOF1-IFS3...IFS1>

CRC14 and CRC15 are differences?

Is this useful or helpful for you?
Kind regards.

**Comment by d3claes** [comment]
Hi,
I tried experimenting with the py-scripts locally on my Home Assistant. 
I have adapted the scripts for VMB1BLS with VMB1BLE as a base. And it works without problems: the module is automatically recognized, and the functions up, down and stop work without problems.
When I have the time, I will take a look at the other two modules I had mentioned: VMB4RF and VMBPIRO.

I've made a pull request at [https://github.com/thomasdelaet/python-velbus/pull/67](url).
